_toc.yml

```
@@ -5,6 +5,8 @@ format: jb-book
 root: intro
 
 parts:
+  - caption: Cervical Cancer Classification
```

```suggestion
  - caption: Cervical Cancer Classif.
```
That's too long. sorry.